### ISSUE TITLE
Incorrect license in documentation

### ISSUE THREAD

**Issue by pombredanne** [nexB/scancode-toolkit#2168]
See https://scancode-toolkit.readthedocs.io/en/latest/license.html

**Comment by Pushpit07** [comment]
@pombredanne What should be the correct license here?

**Comment by AyanSinhaMahapatra** [comment]
@Pushpit07 Scancode has it's [license](https://github.com/nexB/scancode-toolkit/blob/develop/scancode-toolkit.ABOUT#L3) as `apache-2.0 AND cc-by-4.0 AND other-permissive AND other-copyleft`.

The scancode code/the package as a whole is under [`apache-2.0`](https://github.com/nexB/scancode-toolkit/blob/develop/apache-2.0.LICENSE) and the data, like the license and rules, are now under [cc-by-4.0](https://github.com/nexB/scancode-toolkit/blob/develop/cc-by-4.0.LICENSE), after the change in #2337. The license file in docs should have these stated clearly with the associated links, and maybe a word about the change in data with version info.  

Not sure but I don't think `other-permissive` and `other-copyleft` have to be mentioned.

@pombredanne is that okay?

**Comment by chinyeungli** [comment]
https://scancode-toolkit.readthedocs.io/en/latest/license.html
This link is no longer accessible.
I don't see any where in the RTD has reference about the license for SCTK. Should I add it as a "footer" after the copyright notice?

i.e.
`© Copyright nexB Inc. and others. License: apache-2.0 AND cc-by-4.0.`

**Comment by mjherzog** [comment]
@chinyeungli @DennisClark We need a standard approach across AboutCode projects covering:
- The project license(s). For SCTK we explain the license well in the GH README - https://github.com/aboutcode-org/scancode-toolkit?tab=readme-ov-file#license - and we should mirror this information in RTD. My inclination is to put the license information in the Getting Started section because (1) we believe that clear licensing is important and (2) to avoid burying it at the end of the TOC.
- The license for the RTD documentation itself - this should be in the footer. Our default license for documentation is cc-by-4.0 afaik.

**Comment by TheSangamX** [comment]
Hi everyone,

I was going through this issue and saw that some updates were already made to the RTD home page and footer.
Before starting work, I wanted to confirm:

- Are there still documentation pages (e.g. tutorials or sub-sections) where the license information is incorrect or missing?
- Would you like a follow-up PR to audit and fix remaining references across the docs?

Happy to take this up as a first contribution. Thanks!
